### DISCUSSION TITLE
Does my Pi have to be on a network to analyze sound?

### DISCUSSION THREAD

**Discussion by nicolfamilyfarm** [mcguirepr89/BirdNET-Pi#1268] [Q&A]
I live on 400 acres and have been running my Birdnet Pi just outside my home where it can connect to the Wifi.

If I take it out into the bush where there is no Wifi access and connect it to a battery, will it still continue to process sounds?

I understand that I won't be able to connect to it unless I change the Wifi settings, but if I leave it out in the bush for a week and then bring it home and reconnect it will it have processed all the sounds like it was online while it was not on a network?

**Comment by Svardsten53** [comment]
As long as it has power, it will analyze sounds and store all detections on the memory card. No internet access is required for the analysis itself. Uploading to BirdWeather, if defined, will of course not be possible, so this should be disabled.
